package.json

```diff
@@ -24,6 +24,7 @@
     "@well-known-components/metrics": "^1.1.1",
     "canvas": "^2.7.0",
     "cli-progress": "^3.9.0",
+    "dcl-ops-lib": "^5.13.2",
```

Comment:
Watch out, this one should be inside `.ci/package.json`

Reply:
good catch